components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/dao/ApiMgtDAO.java

```diff
@@ -16976,6 +17039,9 @@ public void restoreAPIRevision(APIRevision apiRevision) throws APIManagementExce
                             // Adding product id to uri template id just to store value
                             uriTemplate.setId(rs.getInt(7));
                         }
+                        //Add Operation Endpoint Mapping to URI TEMPLATE
+                        setOperationEndpointMapping(uriTemplate, rs.getInt("URL_MAPPING_ID"));
```

Comment:
Can we make this a constant? "URL_MAPPING_ID"

Reply:
This is the DB fetching so we do not need to worry about this. doesn't it?